addons/mail/static/src/core/common/chat_window_service.js

```diff
@@ -24,6 +24,19 @@ export class ChatWindowService {
         this.ui = services.ui;
     }
 
+    open(thread, replaceNewMessageChatWindow) {
```

Comment:
We could @type this function since there is diff on it now.

Reply:
I'd do if I had other changes to do, but as it's the only minor comment I prefer if the PR moves forward.

We can always make follow up to add missing types, we don't need to wait for diff on the methods to do it.